packages/app/config.d.ts

```diff
@@ -6,6 +6,11 @@ export interface Config {
      * @visibility frontend
      */
     proxyPath?: string;
+    /**
+     * Name of the Backstage flavor (e.g. backstage, rhdh, rhtap)
+     * @visibility frontend
+     */
+    flavor?: string;
```

Comment:
```suggestion
    flavor?: "backstage" | "rhdh" | "rhtap" | (string & {});
```

I'd suggest making this change because it autocompletes `"backstage" | "rhdh" | "rhtap"` while still allowing any generic string.

Reply:
Makes sense, thank you

Reply:
@dzemanov @schultzp2020, It looks like this change is causing `yarn build` to fail with the following:

```
Error: Could not resolve values for path:"type". They are probably incompatible. Values: 
"object"
"string"
```